warp10/src/main/java/io/warp10/script/functions/GEOCELLTO.java

```diff
@@ -0,0 +1,59 @@
+//
+//   Copyright 2020  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.functions;
+
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
+
+/**
+ * Converts a geo cells long to a String HHCode prefix which resolution is implicitly given by its length.
+ * It cannot convert to a byte[] representation because they are limited to resolutions multiple of 4.
+ */
+public class GEOCELLTO extends NamedWarpScriptFunction implements WarpScriptStackFunction {
+
+  public GEOCELLTO(String name) {
+    super(name);
+  }
+
+  @Override
+  public Object apply(WarpScriptStack stack) throws WarpScriptException {
+    Object top = stack.pop();
+
+    if (!(top instanceof Long)) {
+      throw new WarpScriptException(getName() + " expects a " + TYPEOF.TYPE_LONG + " GeoCells.");
+    }
+
+    long geocell = (Long) top;
```

Comment:
This is kind of ugly, mixing a cast and autoboxing. Better to explicitely do ((Long) top).longValue()